core/src/main/java/hudson/Functions.java

```diff
@@ -467,6 +468,15 @@ public static boolean isWindows() {
     public static String printLogRecord(LogRecord r) {
         return formatter.format(r);
     }
+    
+    /**
+     * Get the URL path to the Install Wizard JavaScript.
+     * @return The URL path to the Install Wizard JavaScript.
+     * @since FIXME
+     */
+    public static String getInstallWizardJS() {
```

Comment:
Probably we can avoid dumping new methods like this into the public API. Either use `@Restricted`, or just use a Groovy view which can call the underlying function anyway.